### PR TITLE
Add cURL examples for Auth

### PR DESCRIPTION
**Pull request by victorg1991**
Do you know if this is possible to use cURL to do the 3rd party logins? I didn't add those examples

### CODE REVIEW

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 1]
```diff
@@ -352,6 +395,13 @@ WeDeploy
   .signOut()
   .execute();
 ```
+```text/x-sh
+curl -X POST https://auth-apiswift.wedeploy.io/oauth/revoke \
```
Comment (zenorocha) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: `auth-apiswift` > `<serviceID>-<projectID>`

File: node/electric/src/pages/docs/auth/manage-users/index.md [hunk 2]
```diff
@@ -399,6 +459,18 @@ WeDeploy
     // Not updated.
   });
 ```
+```text-sh
```
Comment (zenorocha) [inline node/electric/src/pages/docs/auth/manage-users/index.md]: `text-sh` > `text/x-sh`

**Comment by victorg1991** [comment]
Done :D

**Pull request merged**